Source/UIKit additions/SVGKLayeredImageView.m

```diff
@@ -75,28 +79,33 @@ - (void)populateFromImage:(SVGKImage*) im
 	{
 #ifndef SVGK_DONT_USE_EMPTY_IMAGE_PLACEHOLDER
         SVGKitLogWarn(@"[%@] WARNING: you have initialized an [%@] with a blank image (nil). Possibly because you're using Storyboards or NIBs which Apple won't allow us to decorate. Make sure you assign an SVGKImage to the .image property!", [self class], [self class]);
-		
+#if SVGKIT_UIKIT
 		self.backgroundColor = [UIColor clearColor];
+#else
+        self.layer.backgroundColor = [NSColor clearColor].CGColor;
+#endif
         
```

Comment:
Moving these constant string into utils. Because now there are multiple files need this...No need to write twice.